File: flutter/rsvp/lib/Models/Response.dart
class Response {
  String _eventLink;
  int _id;
  String _respondentName;
  bool _attending;

  Response(
      String eventLink,
      int id,
      String respondentName,
      bool attending) {
    _eventLink = eventLink;
    _id = id;
    _respondentName = respondentName;
    _attending = attending;
  }


  String get eventLink => _eventLink;

  set eventLink(String eventLink) {
    _eventLink = eventLink;
  }


  int get id => _id;

  set id(int id) {
    _id = id;
  }


  String get respondentName => _respondentName;

  set respondentName(String respondentName) {
    _respondentName = respondentName;
  }


  bool get attending => _attending;

  set attending(bool attending) {
    _attending = attending;
  }

}

File: service/src/main/java/edu/seattleu/rsvp/service/dtos/Response.java
package edu.seattleu.rsvp.service.dtos;

import java.io.Serializable;
import edu.seattleu.rsvp.service.entities.ResponseEntity;

public class Response implements Serializable
{
    private long id;
    private String event_link;
    private String respondentName;
    private boolean attending;

    public Response()
    {
    }

    public Response(long id, String event_link, String respondentName, boolean attending)
    {
        this.id = id;
        this.event_link = event_link;
        this.respondentName = respondentName;
        this.attending = attending;
    }

    public static Response createFromEntity(ResponseEntity responseEntity)
    {
        return null;
    }

    public long getId()
    {
        return id;
    }

    public String getEvent_link()
    {
        return event_link;
    }

    public String getRespondentName()
    {
        return respondentName;
    }

    public boolean isAttending()
    {
        return attending;
    }
}
